great_tables/_gt_data.py

```diff
@@ -89,6 +90,16 @@ def from_data(
             _options=Options(),
         )
 
+        if id is not None:
```

Comment:
Could this happen directly above (on line 90), when the Options object is initially created?

Reply:
Now done, thanks for pairing on this.